src/fitnesseMain/badCodeToTestCheckstyle.java

```diff
@@ -0,0 +1,54 @@
+package fitnesseMain;
+
+import java.util.*; //BAD: star imports
+
+class  Hello
+{
+  public final static String hello = "helloWorld!"; //BAD: modifier order
+
+  public String badCommaWhiteSpaceBeforeAndAfter(String input ,String anotherInput)
+    {
+        return "badCommaWhiteSpaceBeforeAndAfter";
+    }
+
+    public void badGenericsWhiteSpace()
+    {
+      List< String> strings = Arrays.asList("hello", "world");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '<' is followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.GenericWhitespaceCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The String "hello" appears 3 times in the file. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3AMultiple_String_Literals_Custom)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The String "world" appears 2 times in the file. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3AMultiple_String_Literals_Custom)